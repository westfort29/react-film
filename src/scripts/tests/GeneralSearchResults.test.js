import React from 'react';
import { GeneralSearchResults } from '../';
import renderer from 'react-test-renderer';

const SEARCH_SORTING_TYPES =[
  {name: 'release date', isActive: true, id: 'release_date'},
  {name: 'rating', isActive: false, id: 'vote_average'}
];

test('GeneralSearchResults should contain right data', () => {
  const component = renderer.create(
    <GeneralSearchResults
      amount={ 8 }
      searchSortingTypes={SEARCH_SORTING_TYPES}
      onSortOptionClick={ ()=>{} }
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

