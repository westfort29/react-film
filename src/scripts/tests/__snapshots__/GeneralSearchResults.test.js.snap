// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GeneralSearchResults should contain right data 1`] = `
<div
  className="general-search-results"
>
  <div
    className="general-search-results__movies-found"
  >
    8
     Movies found
  </div>
  <div
    className="general-search-results__sorting-options"
  >
    Sort by:
    <button
      className="general-search-results__sorting-button general-search-results__sorting-button--active"
    >
      release date
    </button>
    <button
      className="general-search-results__sorting-button"
      onClick={[Function]}
    >
      rating
    </button>
  </div>
</div>
`;
